docs/connect.html

```diff
@@ -583,6 +644,10 @@ <h5><a id="connect_connectorexample" href="#connect_connectorexample">Connector
 public class FileStreamSourceConnector extends SourceConnector {
     private Map&lt;String, String&gt; props;</pre>
 
+    <p>Add a corresponding ServiceLoader manifest to your resources in <code>META-INF/services/org.apache.kafka.connect.source.SourceConnector</code> to allow it to be discovered by the runtime.</p>
```

Comment:
This kind of breaks the flow of writing our connector class; it comes between the `public class FileStreamConnector...` declaration and the `public Class<? extends Task> taskClass()` declaration.

Maybe we can put this before the class declaration, with a note that the name of the connector we're developing will be `FileStreamSourceConnector`?

Also, we should probably link to the docs on plugin discovery modes in this section. Maybe something like "See <a href="#connect_plugindiscovery">plugin discovery</a> for more info on this step"?